### PR TITLE
Consistency is not everything - but it is a worthy goal.

### PR DESCRIPTION
**Pull request by csdodd**
We should aim to be consistent with the repo, and be able to use our common sense.

### CODE REVIEW

**Comment by designatednerd** [comment]
@csdodd You said some helpful things in the meeting today around this, particularly around within-repo consistency. Would you mind throwing that stuff in here?

**Comment by csdodd** [comment]
Heh, was hoping to get away with doing the smallest amount of work possible!

Makes sense though, will update!

**Comment by csdodd** [comment]
Updated!

I decided to introduce the playbook and its purpose at the top of `README`

**Review by elland — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ![logo](https://raw.githubusercontent.com/bakkenbaeck/iOS-playbook/master/assets/logo-v1.png)
 
-Don't forget to check out our favorite [suite of components and helper methods](http://github.com/usesweet) and the [other B&B iOS repos](https://github.com/bakkenbaeck?language=swift) to see if we've already built something to help with what you're trying to do.
+This playbook consists of a set of guidelines that may be useful to you. It is a living document that contains good advice on how to style code, communicate, use git, and more. As a guideline, it is not a rule of law. Being consistent with the repo and your fellow developers takes precedent over this playbook. Also, the playbook can be ignored if a developer believes that adhering to the guidelines would give a worse result. At which point - you may want to consider updating the playbook 😀
```
Comment (designatednerd) [inline README.md]: IIRC it's `takes precedence` rather than `takes precedent`

**Review by designatednerd — APPROVED** [review]
One grammar nerd thing otherwise 👍

**Pull request merged**